### PR TITLE
Add LLPoint test

### LINKED ISSUE #26: [quality] - Code Testing Required
Test suite and automated testing required for PRs

### PR DESCRIPTION
**Pull request by Brett-Parker**
fixes #26 

Only a shell to begin with but distance, angle and nearest are tested. This is only single test and really there should be tests for edge cases too. Step 1 is tests.

### CODE REVIEW

File: tests/test_Point.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+"""Test for LLPoint.py"""
```
Comment (dubstar-04) [inline tests/test_Point.py:1]: I think this should be inside the class

File: tests/test_Point.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import os
+import sys
+import unittest
+
+thisFolder = os.path.dirname(os.path.abspath(__file__))
+parentFolder = os.path.dirname(os.path.dirname(thisFolder))
+sys.path.append(parentFolder)
+from LibLathe.LLPoint import Point
```
Comment (dubstar-04) [inline tests/test_Point.py:10]: try and keep the imports together i.e move above line 7
